taxi/views.py

```diff
@@ -1,3 +1,14 @@
 from django.shortcuts import render
+from taxi.models import Driver, Manufacturer, Car
```

Comment:
Separate 3rd party imports and local imports with one blank line according to PEP-8.